src/dist/block_service/fds/fds_service.cpp

```diff
@@ -95,10 +98,33 @@ DEFINE_THREAD_POOL_CODE(THREAD_POOL_FDS_SERVICE)
 DEFINE_TASK_CODE(LPC_FDS_CALL, TASK_PRIORITY_COMMON, THREAD_POOL_FDS_SERVICE)
 
 const std::string fds_service::FILE_LENGTH_CUSTOM_KEY = "x-xiaomi-meta-content-length";
-const std::string fds_service::FILE_LENGTH_KEY = "content-length";
 const std::string fds_service::FILE_MD5_KEY = "content-md5";
 
-fds_service::fds_service() {}
+fds_service::fds_service()
+{
+    const int BYTE_TO_BIT = 8;
+
+    uint64_t sst_file_size = dsn_config_get_value_uint64("pegasus.server",
```

Comment:
There is an other config you should notice, that is `write_buffer_size`, it is the memtable size threashold flushed to level 0, which is also contained in backup directory. It's 64MB by default, but in Pegasus, it will be random in 64MB*[0.75, 1.25].
https://github.com/XiaoMi/pegasus/blob/master/src/server/pegasus_server_impl.cpp#L2420

Reply:
In BULK_LOAD scenario, it's 64MB*4*[0.75, 1.25], although it's rare to backup a table when it's in BULK_LOAD scenario mode, but we could provide a way observe when it happend, for example, add a log, add a metric would be better.